README.md

```diff
@@ -1,65 +1,131 @@
-FoxBox
-======
+# FoxBox
 
 [![Build Status](https://travis-ci.org/fxbox/foxbox.svg?branch=master)](https://travis-ci.org/fxbox/foxbox)
 [![License](https://img.shields.io/badge/license-MPL2-blue.svg)](https://raw.githubusercontent.com/fxbox/foxbox/master/LICENSE)
 
 
-## Target
+## Technologies
 
-Target hardware for prototyping is the Raspberry Pi 2. It's ARMv7 with Neon, but building without Neon support should do for now. Target OS is the latest Raspbian which is based on Debian 8.0 Jessie.
+### Rust
 
+We're using Rust for the daemon/server. Currently 1.8.x nightly is required. We
+plan to track v1.8.x into stable and then continue using stable releases from
+then on. As of 2016-02-04:
```

Comment:
Fair enough. I opened #32 and only heard from Hub, which sounded liked a decent plan.

It's easy to remove "We plan to track v1.8.x into stable and then continue using stable releases from then on.". Would you like to mention/address our intent on try getting on stable at some point?